site/content/blog/gsoc-2020-accepted-projects-announcement.md

```diff
@@ -0,0 +1,32 @@
++++
+date = "2020-05-04T18:01:00+00:00"
+title = "GSoC Accepted Projects Announcement"
+author = "Sam Elliott and Pirmin Vogel"
++++
+
+We are pleased to announce that we will be mentoring two students as part of Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/).
+
+We are looking forward to working with Flavien and Yuichi on features and tools to improve IP such as Ibex, our open-source RISC-V core.
+
+## Flavien Solt: Simulated Memory Controller
+
+It is a common pitfall to misinterpret or incorrectly scale performance numbers derived from benchmarks run on an FPGA-based SoC design.
+The problem is that your external memory interface is running at a very high speed compared to the core CPU (e.g. a 25MHz core clock speed but external memory running a several hundred MHz).
+This can be misleading when trying to consider what the performance would be on an ASIC, as the CPU clock speed could be many times higher but the memory frequency remains the same or increases by a much smaller amount.
+The solution is to have a simulation-ready memory controller that will produce delays more representative of a system where the memory interface is running at a much slower speed.
```

Comment:
have a simulation-ready => implement a "simulated"

Reply:
Actually Rob suggests instead 'The solution is to implement a memory controller for the FPGA that can accurately "simulate" memory access delays in terms of the processor's clock cycles.'

Reply:
I went for Rob's solution.